variables.tf

```diff
@@ -86,6 +86,26 @@ variable "databricks_cluster_node_type" {
 
 # Security
 
+variable "application_id" {
```

Comment:
Provide default values for this and the other variables to that it's not a required argument for the module